fusesoc/main.py

```diff
@@ -702,6 +708,7 @@ def parse_args():
         "--sync-type",
         help="The provider type for the library. Defaults to 'git'.",
         choices=["git", "local"],
+        dest="sync-type",
```

Comment:
Added dest as this argument was being set as `sync_type` by the parser so I don't think it would be used